src/plugins/dataflow-tool/nodes/controls/dropdown-list-control.tsx

```diff
@@ -131,15 +133,17 @@ export class DropdownListControl extends Rete.Control {
       readonly,
       value: initial,
       onItemClick: (v: any) => {
-        this.emitter.trigger("selectnode", { node: this.getNode() });
+        this.emitter.trigger("selectnode", { node: this.getNode()});
```

Comment:
Super nitpicky, but seems strange to remove the trailing space but leave the leading space.

Reply:
good 👀 !